src/runtime/native/GeneratedTypeOffsets.cs

```diff
@@ -7,6 +7,9 @@ namespace Python.Runtime.Native
     [StructLayout(LayoutKind.Sequential)]
     abstract class GeneratedTypeOffsets
     {
+        // XXX: Use field to trick the public property finding
+        public readonly int Size;
+
```

Comment:
Why do you think this is better, than the single line you removed from ABI.cs? This is hacky and uses reflection.

Reply:
It's not related to that line, the reason for removing that line just because was because that's not the fixed slot anymore.
Well, it was just need something to store the size, tp_basicsize might capable do that too, this part can be improved.

Reply:
This field has the same purpose and value as the one you removed from ABI - `sizeof(PyTypeObject)`